### ISSUE TITLE
How to display line breaks in annotations?

### ISSUE THREAD

**Issue by shanehou** [openannotation/annotator#605]
I found that annotatorjs escaped `<` and `>` before it put annotations into the viewer. But I need to display at least line breaks in my annotations.

I'm thinking about two methods.
### First method:

set

```
<style>
    .annotator-listing {
        white-space: pre-wrap;
    }
</style>
```

after loading the annotator's CSS file and use `\n` in the annotations to display line breaks.
### Second method:

Add a new plugin to replace back `<` and `>`:

```
Annotator.Plugin.Unescape = function(element) {
    return {
        pluginInit: function() {
            this.annotator.subscribe("annotationViewerTextField", function(field, annotation) {
                $(field).html(annotation.text);
            });
        }
    }
};
```

But I think neither method is good enough. Is there any way to do this?

**Comment by tilgovi** [comment]
This could clash with other plugins, like the markdown plugin. Is it possible you have that enabled, too?

**Comment by shanehou** [comment]
No, I didn't use those plugins. I think at least line breaks should be supported.
